lib/environment.rb

```diff
@@ -416,6 +416,9 @@ def service_project
       unless @service_project
         project = Project.new(name: "tests-" + EXECUTOR_NAME.downcase, env: self)
         unless project.exists?
+          unless project.active?(cached: true)
+            project.wait_to_disappear(admin)
```

Comment:
we might not need this, or maybe it should be `project.ensure_deleted(user: nil)` as well.

Reply:
I guess `phase:` is either `Active` or `Terminating` for namespaces, so we might not need `ensure_deleted` because `!active?` means it is `Terminating`.